packages/esbuild-plugin-pnp/sources/index.ts

```diff
@@ -0,0 +1,61 @@
+import {PnpApi}                                                                        from '@yarnpkg/pnp';
+import {OnLoadArgs, OnLoadResult, OnResolveArgs, OnResolveResult, Plugin, PluginBuild} from 'esbuild-wasm';
+import fs                                                                              from 'fs';
+
+const defaultExtensions = [`.js`, `.jsx`, `.ts`, `.tsx`, `.json`];
```

Comment:
Should probably pick up `.mjs` as well so it can treeshake clipanion

Reply:
Although technically we don't want to treeshake clipanion, since it's a dynamic library that plugins consume.

Reply:
True, for that dependency it's more that the esm output is smaller than the cjs one